packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -145,6 +145,13 @@ export default function EditPostPreferencesModal() {
 									label={ __( 'Display block breadcrumbs' ) }
 								/>
 							) }
+							<EnableFeature
+								featureName="strictColorContrastChecks"
+								help={ __(
+									'Makes the color contrast checker more strict, to comply with WCAG AAA guidelines.'
```

Comment:
It's not clear that by default it's checking AA — I can see a user confused that if you disable it you wouldn't receive warnings.

Maybe we can render a dynamic value "Currently checking for AA/AAA" based on the state of the toggle?